packages/jellyfish-testing/src/anchor.ts

```diff
@@ -0,0 +1,21 @@
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { TestingGroup } from './testing'
+
+export class TestingAnchor {
+  constructor (
+    private readonly container: MasterNodeRegTestContainer,
+    private readonly rpc: JsonRpcClient
+  ) {
+  }
```

Comment:
I think you can add this into `TestingGroup`

```suggestion
export class TestingGroupAnchor {
  constructor (
    private readonly testingGroup: TestingGroup,
  ) {
  }
```